descriptors/ModuleDescriptor-template.json

```diff
@@ -555,22 +556,22 @@
       ]
     },
     {
-      "permissionName": "orders.acquisitions-units-assignments.item.post",
+      "permissionName": "orders.acquisitions-units-assignments.assign",
       "displayName": "Acquisitions unit assignment - create unit assignment",
-      "description": "Create a new acquisitions unit assignments"
+      "description": "Assign new order to acquisitions units"
```

Comment:
should this be assign acq units to new order

Reply:
I thought we were assigning an order to a unit, and not vice versa.

Reply:
@KVupp I'm going to merge PR to get the changes to testing env.  I will change the description in the following PR for MODORDERS-254/MODORDERS-255, if needed.